bugmon_tc/common/__init__.py

```diff
@@ -62,18 +71,18 @@ def fetch_artifact(task_id, artifact_path):
     return response
 
 
-def fetch_json_artifact(task_id, artifact_path):
+def fetch_json_artifact(task_id: str, artifact_path: Path) -> Dict[str, Any]:
     """Fetch a JSON artifact
 
     :param task_id: Task id
     :param artifact_path: Path to the artifact
     """
     resp = fetch_artifact(task_id, artifact_path)
-    return resp.json()
+    return cast(Dict[str, Any], resp.json())
 
 
 @contextmanager
-def fetch_trace_artifact(artifact_path):
+def fetch_trace_artifact(artifact_path: Path) -> Generator[Path, None, None]:
```

Comment:
You can use `Iterator[Path]` instead like in `BugMonitorTask.fetch_bugs`. Either is fine, but you should be consistent.